float64 head_x
float64 head_y
float64 head_dx
float64 head_dy
float64 body_dx
float64 body_dy
uint64 framenumber
float64 processing_time

float64 NO_DETECT=1000000

